apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -258,6 +258,21 @@ export default function SectionsSetUpContainer({
       });
   };
 
+  const consolidatedCourseData = () => {
+    if (sections[0].courseOfferingId !== null) {
+      return {
+        courseOfferingId: sections[0].courseOfferingId,
+        versionId: sections[0].courseVersionId,
+        unitId: sections[0].unitId,
+        hasLessonExtras: sections[0].lessonExtras,
+        hasTextToSpeech: sections[0].ttsAutoplayEnabled,
+        displayName: sections[0].courseDisplayName,
+      };
+    } else {
+      return null;
+    }
+  };
+
```

Comment:
@lfryemason let me know your thoughts on this... we still have this issue with the data getting passed through (a different issue, but still connected to the data).  The data expected by this component is nested - I thought of arranging the incoming data directly here.  This muddies up this component a bit (and if we go this direction, a comment would be needed), but I don't really want to mess with the data coming out of the redux store.  Thoughts?

Reply:
Could you explain this a bit more, I'm not sure I understand why there is a difference here or what data is nested.

Reply:
I will also say, I think it is OK to say "that's a later problem" and ONLY merge in a PR that shows the edit page with incomplete data and handle the data issue in a separate PR.   I threw this in here to get your ideas on it a bit and model what A solution could look like.